src/command_modules/azure-cli-billing/tests/test_billing_commands.py

```diff
@@ -0,0 +1,49 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import (ScenarioTest, JMESPathCheck)
+
+
+class AzureBillingServiceScenarioTest(ScenarioTest):
+    def _validate_invoice(self, invoice, includeUrl=False):
+        assert invoice
```

Comment:
Please avoid using `assert` operator. Instead use `self.assertXXX` series of methods. The later methods offer better output during assertion failures.